sql/columndefault.go

```diff
@@ -162,3 +181,38 @@ func (e *ColumnDefaultValue) CheckType(ctx *Context) error {
 	}
 	return nil
 }
+
+type UnresolvedColumnDefault struct {
+	exprString string
+}
+
+func (u UnresolvedColumnDefault) Resolved() bool {
+	return false
+}
+
+func (u UnresolvedColumnDefault) String() string {
+	return u.exprString
+}
+
+func (u UnresolvedColumnDefault) Type() Type {
+	panic("UnresolvedColumnDefault is a placeholder node, but Type() was called")
+}
+
+func (u UnresolvedColumnDefault) IsNullable() bool {
+	return true
+}
+
+func (u UnresolvedColumnDefault) Eval(ctx *Context, row Row) (interface{}, error) {
+	panic("UnresolvedColumnDefault is a placeholder node, but Eval() was called")
```

Comment:
Is this a panic or just an error? Something has gone wrong, but is it a panic-level something going wrong?

Reply:
It's akin to the behavior for unresolved column, if we hit this it's a very serious problem that needs attention